test/src/math/test_vecmat.cpp

```diff
@@ -35,7 +35,7 @@ class VecmatTest : public test::FSTestFixture {
 	VecmatTest() : test::FSTestFixture() { pushModDir("vecmat"); }
 
 protected:
-	void SetUp() override { test::FSTestFixture::SetUp(); }
+	void SetUp() override { test::FSTestFixture::SetUp(); srand(1); }
```

Comment:
Please format this so that the two statements are on different lines. The clang-format config should be mostly correct so you can also use that to format this line if you want.